tests/lib/gladecheck.py

```diff
@@ -28,24 +28,16 @@
     print("No module named lxml, you need to install the python3-lxml package")
     raise
 
-try:
-    from pocketlint.translatepo import translate_all
-except ImportError:
-    print("Unable to load po translation module. You may need to install python3-polib")
-    raise
-
 from abc import ABCMeta, abstractmethod
-import os
 import unittest
-import copy
-# FIXME: Enable glade tests back when we found a way how to migrate them to python3 unittest
-# pylint: disable=import-error
-import nose
 
 from filelist import testfilelist
 
 import logging
-log = logging.getLogger('nose.plugins.glade')
+log = logging.getLogger('unittest')
+
+glade_trees = []
```

Comment:
I would just set `glade_trees = load_glade_trees_from_files()`, simplify the `load_glade_trees_from_files` function  and remove `gladecheck.load_glade_trees_from_files()` from above.